### PR TITLE
Ww 442 fix sdk issues

### PR DESCRIPTION
**Pull request by NateDolz**
[ww-442](https://spurjob.atlassian.net/browse/WW-442?atlOrigin=eyJpIjoiZWVjOWJjZTA5MWE3NDAyODgwZDA2MWNlNDc0YzIwOTkiLCJwIjoiaiJ9)

### CODE REVIEW

File: src/SpurClient.php [hunk 1]
```diff
@@ -563,33 +563,33 @@ public function removeWorker(int $job_id, int $worker_id)
...
     {
-        return $this->get("places/{$place_id}/limited_employees/{$employee_id}");
+        return $this->get("api/places/{$place_id}/limited_employees/{$employee_id}");
     }
 
-    public function getLimitedEmployeesFromLocation(int $location_id, $params)
+    public function getLimitedEmployeesFromLocation(int $location_id, array $params = [])
     {
-        return $this->get("locations/{$location_id}/limited_employees", $params);
+        return $this->get("api/ocations/{$location_id}/limited_employees", $params);
```
Comment (ThomDavis) [inline src/SpurClient.php]: some good `ocaltions` in this PR

Comment (ninjaparade) [reply src/SpurClient.php]: are `ocations` the same as `laces` in the api?

Comment (ninjaparade) [reply src/SpurClient.php]: are there any `ccounts`?

Comment (ninjaparade) [reply src/SpurClient.php]: should this be `$ocation_id`?

Comment (NateDolz) [reply src/SpurClient.php]: 🙃

**Pull request merged**